go/kbfs/simplefs/simplefs.go

```diff
@@ -2268,6 +2276,69 @@ func (k *SimpleFS) SimpleFSSetFolderSyncConfig(
 	return err
 }
 
+// SimpleFSSyncConfigAndStatus implements the SimpleFSInterface.
+func (k *SimpleFS) SimpleFSSyncConfigAndStatus(
+	ctx context.Context) (res keybase1.SyncConfigAndStatusRes, err error) {
+	bytesAvail, bytesTotal := k.getLocalDiskStats(ctx)
+	tlfIDs := k.config.GetAllSyncedTlfs()
+
+	session, err := idutil.GetCurrentSessionIfPossible(
+		ctx, k.config.KBPKI(), true)
+	if err != nil {
+		return keybase1.SyncConfigAndStatusRes{}, err
+	}
+
+	res.Folders = make(
+		[]keybase1.FolderSyncConfigAndStatusWithFolder, len(tlfIDs))
+
+	for i, tlfID := range tlfIDs {
+		config, err := k.config.KBFSOps().GetSyncConfig(ctx, tlfID)
+		if err != nil {
+			return keybase1.SyncConfigAndStatusRes{}, err
+		}
+
+		if config.Mode == keybase1.FolderSyncMode_DISABLED {
+			// Shouldn't happen.
```

Comment:
Since this shouldn't happen, do we not care that the output will look a little strange if this happens?
Maybe we should at least log something?

Reply:
I turned it into a panic just to make sure we find out if it ever happens.